docs/MAINTAINING.md

```diff
@@ -36,6 +37,18 @@ Here are the milestones we use and how we use them:
 
 With this flow, any un-assigned issues are essentially in triage state and once triaged are either "Backlog" or "Backlog Candidates". They will eventually move to "On Deck" (or be closed). Lastly, they will end up on a version milestone where they will be worked on.
 
+### Triage
+
+We use the following process for triaging GitHub issues:
+
+1. issue comes in
+2. add appropriate labels
+   1. if we need to look further into it, add "needs-investigation"
+3. add to a milestone
+   1. if it should be fixed soon, add to version milestone or "On Deck"
```

Comment:
Maybe worth spelling out more explicitly what "On Deck" means here, especially for non-native speakers that are not familiar with that colloquialism. Alternatively, we can rename it to "Upcoming" or "Next" or something?

Reply:
We went with "On Deck" because that's what vscode uses. Though I see your point too. And it is described under Milestone 🤔 

Unless you feel strongly, I think it's probably okay?

Reply:
Yea it's fine as it is! Was just thinking out loud, all good